### PR TITLE
Add import model arguments

### PR DESCRIPTION
**Pull request by binary-person**
Add feature of importing weights by arguments:
python3 predict.py insert_weights_file_here.hdf5

### CODE REVIEW

File: predict.py [hunk 1]
```diff
@@ -67,7 +67,11 @@ def create_network(network_input, n_vocab):
     model.compile(loss='categorical_crossentropy', optimizer='rmsprop')
 
     # Load the weights to each node
-    model.load_weights('weights.hdf5')
+    try:
+        model.load_weights(sys.argv[2])
```
Comment (Skuldur) [inline predict.py]: Make this a named variable. Otherwise it might confuse some people.

File: predict.py [hunk 2]
```diff
@@ -67,7 +67,11 @@ def create_network(network_input, n_vocab):
...
+        print('Loaded weights from argument')
+    except NameError:
+        model.load_weights('weights.hdf5')
```
Comment (Skuldur) [inline predict.py]: Make it load new_weights.hdf5 instead of weights.hdf5 by default, I've been meaning to fix that but haven't gotten around to it so this would be the perfect place to throw that change in.

File: lstm.py [hunk 3]
```diff
@@ -85,22 +86,25 @@ def prepare_sequences(notes, n_vocab):
...
-    model.add(LSTM(512, return_sequences=True))
-    model.add(Dropout(0.3))
-    model.add(LSTM(512))
-    model.add(Dense(256))
-    model.add(Dropout(0.3))
-    model.add(Dense(n_vocab))
-    model.add(Activation('softmax'))
-    model.compile(loss='categorical_crossentropy', optimizer='rmsprop')
+    try:
+        model = load_model(sys.argv[2])
```
Comment (Skuldur) [inline lstm.py]: I would like you to revert this change. The model of the repository should be the one in the article, for consistency. Also I'm going to do a minor update to the article and repository soon, which will improve the model slightly.

**Comment by Gnu-Bricoleur** [comment]
Hi, I'm sorry i really should have checked the other pull request before submitting mine, I'm still a bit new to this and i just wanted to contribute as I'm starting to play with your code.
I can modify my pull request to add the functionalities you requested from binary-person if he is not around to do it on his own pull request;

**Comment by binary-person** [comment]
I might make another pull request that is up-to-date with the current model. Right now, this pull request is obsolete
